packages/components/card/README.md

```diff
@@ -534,7 +533,9 @@ The card Nunjucks macro takes the following arguments:
 | **headingLevel**    | integer | No       | Optional heading level for the card heading. Default: 2                                                             |
 | **href**            | string  | No       | The value of the card href attribute                                                                                |
 | **clickable**       | boolean | No       | If set to true, then the class `nhsuk-card--clickable` will be applied.                                             |
+| **arrow**           | boolean | No       | If set to true, then the class `nhsuk-card__content--arrow` will be applied.                                        |
```

Comment:
Is this 'nhsuk-card__content--arrow' class used anywhere?

Reply:
That needed updating to reflect the new naming convention of primary card; I've updated that now.